Contributors.md

```diff
@@ -1,3 +1,4 @@
 # Contributors
 
 **Kyle Kauzlarich** https://github.com/kyle-kauzlarich
+**Toby Glaze ** https://github.com/Stryyder
```

Comment:
The bolded font will work once you remove the space between your last name and the trailing **